### PR TITLE
Upgrade to latest versions 

### PR DESCRIPTION
**Pull request by rajadilipkolli**
- separated workflows for ui and service
- upgraded to SB 3
- added few UI tests
- added few instructions

### CODE REVIEW

File: techstack.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
+<!--
+&lt;--- Readme.md Snippet without images Start ---&gt;
+## Tech Stack
+rajadilipkolli/springboot-kubernetes-youtube-series is built on the following main stack:
```
Comment (sivaprasadreddy) [inline techstack.md:4]: We can remove the github username `sivaprasadreddy` or `rajadilipkolli` to be generic.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Code for "SpringBoot + Kubernetes Tutorial" YouTube Series https://www.youtube.c
 ## How to run?
 
 ```shell
-$ git clone https://github.com/sivaprasadreddy/springboot-kubernetes-youtube-series.git
```
[the remark this answers is not in the document]
Comment (rajadilipkolli) [reply README.md]: resolved

File: techstack.md (hunk 1 above)
Comment (rajadilipkolli) [reply techstack.md]: Fixed

**Comment by rajadilipkolli** [comment]
Hi @sivaprasadreddy , fixed all review comments.  Can you please review and merge, I suggest before merging take a backup of current code and add a comment in ReadMe. so that if users are coming directly from YouTube they will still have that reference.

**Review by 544609380 — COMMENTED** [review]
Ok

**Pull request merged**